.tools/readmes/multi.py

```diff
@@ -50,6 +51,13 @@ def main():
         default=True,  # Change this to default false when we're ready to use this generally.
```

Comment:
Why not now?

Reply:
Not in this PR, but yeah, I'll bring it up in standup.